New Spotlights Season: Summer 2020 (#3826)
ppy/osu-wiki: news/2020-07-06-new-beatmap-spotlights-season-summer-2020.md

---
layout: post
title: "New Beatmap Spotlights Season: Summer 2020"
date: 2020-07-06 07:00:00 +0000
---

For the first time since our initial announcement, the renewed Beatmap Spotlights boot off with its first Summer Season and a plethora of rewards for everybody joining the brand-new competitive Seasonal League.

![](/wiki/shared/news/banners/beatmap-spotlights.jpg)

> Everybody in our new team of curators worked hard to bring forth the most excellent maps osu! has to offer and to present you a new competitive league to enjoy their selections. While the first ideas have been kept vague in our first announcement, here are some details on how this new system works:

Reviewer: > Everybody in our new team of curators worked hard to bring forth the most excellent maps osu! has to offer and to present you[...]
Comma before "and" makes this easier to read/sound better imo

> Everybody in our new team of curators worked hard to bring forth the most excellent maps osu! has to offer and to present you a new competitive league to enjoy their selections. While the first ideas have been kept vague in our first announcement, here are some details on how this new system works:

Reviewer: ```suggestion
Everybody in our new team of curators worked hard to bring forth the most excellent maps osu! has to offer and to present you a new competitive league to enjoy their selections. While the first ideas have been kept vague in our first announcement, below are some details on how this new system works:
```
"here" creates an incomplete clause

> - **Every weekend in this season we will be hosting a weekly timeshift lobby using a playlist created from the pool of curated beatmaps using osu!lazer.** Therefore, you have to join the timeshift lobbies on osu!lazer in order to participate in the competitive league. If you have never heard of timeshift multiplayer lobbies before, [you can learn more here](https://blog.ppy.sh/enjoying-osu-with-others/), or just [get osu!lazer yourself and try it out now](https://github.com/ppy/osu/releases).

Reviewer: ```suggestion
- **Every weekend in this season, we will be hosting a weekly timeshift lobby using a playlist created from the pool of curated beatmaps using osu!lazer.** Therefore, you have to join the timeshift lobbies on osu!lazer in order to participate in the competitive league. If you have never heard of timeshift multiplayer lobbies before, [you can learn more here](https://blog.ppy.sh/enjoying-osu-with-others/), or just [get osu!lazer yourself and try it out now](https://github.com/ppy/osu/releases).
```

- Every Monday, the week's playlist will be unveiled on the [Beatmap Spotlights wiki page](/wiki/Beatmap_Spotlights). This means that **beatmaps being played in the Spotlights league lobbies will change every week**. For future seasons, we plan to unveil these maps in a livestream as a beatmap showcase.

> - The results of the weekly timeshift lobby will be collated into a big **Seasonal Ranking**. Depending your position on the Seasonal Ranking, you can earn yourself a **unique and permanent profile badge**. The profile badges will already be awarded after the 2nd week of this season and will be updated every week, reflecting your current position in the Seasonal Ranking.

Reviewer: ```suggestion
- The results of the weekly timeshift lobby will be collated into a big **Seasonal Ranking**. Depending on your position in the Seasonal Ranking, you can earn yourself a **unique and permanent profile badge**. The profile badges will already be awarded after the 2nd week of this season and will be updated every week, reflecting your current position in the Seasonal Ranking.
```

- There will be **four playlists in total, each consisting of 5 beatmaps** and each being played three times during the season. This gives you plenty of opportunities to revisit your weekly scores as only your best total score on a playlist will be considered for your position on the Seasonal Ranking.

> - On top of that **the Top 10 people of each timeshift lobby will be awarded with 1 week of supporter tag**.

Reviewer: can we link something to "Top 10" here? seems like it would fit nice

> - At the end of the season, we will provide a **new unlockable medal** for those who want to play through the Seasonal Beatmaps once the season has concluded and **the best mapper of this season will be elected**. More information about that will follow at a later point.

Reviewer: ```suggestion
- At the end of the season,  **the best mapper of this season will be elected**, and we will provide a **new unlockable medal** for those who want to play through the Seasonal Beatmaps once the season has concluded. More information about this will follow at a later point.
```
This sentence flows better imo. "This" is a little ambiguous tho

> Of course each game mode will have their own leaderboard and playlists. If you are interested in the full details of this system, you can visit the [Beatmap Spotlights wiki page](/wiki/Beatmap_Spotlights) to learn more.

Reviewer: ```suggestion
Of course, each game mode will have their own leaderboard and playlists. If you are interested in the full details of this system, you can visit the [Beatmap Spotlights wiki page](/wiki/Beatmap_Spotlights) to learn more.
```
No comma would suggest a longer sentence and a very different intonation

## Season Schedule

> With that being said, the Summer 2020 season has been fully prepared and is ready for deployment. Therefore, the first timeshift lobby will open **this Friday** already. The first Playlist A can be found on the [Beatmap Spotlights wiki page](/wiki/Beatmap_Spotlights) now. Remember that your badge will update every week once they have been initially distributed.

Reviewer: ```suggestion
With that being said, the Summer 2020 season has been fully prepared and is ready for deployment. Therefore, the first timeshift lobby will open **this Friday** already. Playlist A can be found on the [Beatmap Spotlights wiki page](/wiki/Beatmap_Spotlights) now. Remember that your badge will update every week once they have been initially distributed.
```
"The first" seems a bit redundant considering the sentence before

| Event | Time |
| --: | :-- |
| Season Start | Jul 6th 2020 |
| Timeshift Lobby: Playlist A | Jul 10th - Jul 12th 2020 |
| Timeshift Lobby: Playlist B | Jul 17th - Jul 19th 2020 |
| Initial Profile Badge Distribution | Jul 21st - Jul 23rd 2020 |
| Timeshift Lobby: Playlist C | Jul 24th - Jul 26th 2020 |
| Timeshift Lobby: Playlist D | Jul 31st - Aug 2nd 2020 |
| Timeshift Lobby: Playlist A | Aug 7th - Aug 9th 2020 |
| Timeshift Lobby: Playlist B | Aug 14th - Aug 16th 2020 |
| Timeshift Lobby: Playlist C | Aug 21st - Aug 23rd 2020 |
| Timeshift Lobby: Playlist D | Aug 28th - Aug 30th 2020 |
| Timeshift Lobby: Playlist A,B | Sep 4th - Sep 6th 2020 |
| Timeshift Lobby: Playlist C,D | Sep 11th - Sep 13th 2020 |
| Season Conclusion | Sep 14th - Sep 18th 2020 |

## Rewards

> Depending on your position on the Seasonal Ranking, you will be awarded with a unique profile badge. Your exact position will only be known to you when you are in the Top 50. Additionally, the Top 10 players of every timeshift lobby will be awareded with 1 week of supporter tag.

Reviewer 2: ```suggestion
Depending on your position on the Seasonal Ranking, you will be awarded with a unique profile badge. Your exact position will only be known to you when you are in the Top 50. Additionally, the Top 10 players of every timeshift lobby will be awarded with 1 week of supporter tag.
```

> Depending on your position on the Seasonal Ranking, you will be awarded with a unique profile badge. Your exact position will only be known to you when you are in the Top 50. Additionally, the Top 10 players of every timeshift lobby will be awareded with 1 week of supporter tag.

Reviewer: ```suggestion
Depending on your position on the Seasonal Ranking, you will be awarded with a unique profile badge. Your exact position will only be known to you when you are in the Top 50. Additionally, the Top 10 players of every timeshift lobby will be awarded with 1 week of an osu!supporter tag.
```

| ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/gold_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/gold_2.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/gold_3.png) | Gold | 10% - 25% |
| ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/silver_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/silver_2.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/silver_3.png) | Silver | 25% - 50% |
| ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/bronze_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/bronze_2.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/bronze_3.png) | Bronze | 50% - 70% |
| ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/copper_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/copper_2.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/copper_3.png) | Copper | 70% - 95% |
| ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/iron_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/iron_2.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/iron_3.png) | Iron |  95% - 100% |

> This table only shows one of four badge versions. Each game mode has an own badge version.

Reviewer 2: ```suggestion
This table only shows one of four badge versions. Each game mode has its own badge version.
```

## Winter Spotlights results

> We did not forgot the winners of the Winter Beatmap Spotlights 2020. Congratulations to every single one of them! They will receive 3, 2, and 1 month of osu!supporter for 1st, 2nd, and 3rd place respectively for their efforts!

Reviewer: ```suggestion
We did not forget the winners of the Winter Beatmap Spotlights 2020. Congratulations to every single one of them! They will receive 3, 2, and 1 month of osu!supporter for 1st, 2nd, and 3rd place respectively for their efforts!
```

| ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/copper_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/copper_2.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/copper_3.png) | Copper | 70% - 95% |
| ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/iron_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/iron_2.png) ![](/wiki/Beatmap_Spotlights/img/badges/summer_2020/iron_3.png) | Iron |  95% - 100% |

> This table only shows one of four badge versions. Each game mode has an own badge version.

Reviewer: ```suggestion
This table only shows one of four badge versions. Each game mode has its own badge version.
```
I assume this was a typo. Also, this seems like it should be bolded.
